internal/resolvers/twitch/resolver.go

```diff
@@ -17,10 +17,11 @@ import (
 const (
 	twitchClipsTooltipString = `<div style="text-align: left;">
 <b>{{.Title}}</b><hr>
+<b>Author:</b> {{.AuthorName}}<br>
```

Comment:
Personally, I think something like "Clipped by:" would be nicer here. But maybe that's up to personal taste

Reply:
I thought following already common one-word format would be nice, but I guess your suggestion makes more sense, Though I guess I'd leave it up to pajlada to decide which one fits better.

Reply:
Went with "Clipped by" :+1: